exercises/concept-wip/crystal-hunter/.docs/instructions.md

```diff
@@ -0,0 +1,53 @@
+# Instructions
```

Comment:
The instructions are clear and correct, and following them, I was able to easily solve the exercise without looking up any hints and without cheating 👍